src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -242,12 +243,19 @@ const formConfig = {
           uiSchema: uploadPtsdDocuments.uiSchema,
           schema: uploadPtsdDocuments.schema,
         },
-        newUnemploymentFollowUp: {
-          title: 'File a Claim for Individual Unemployability',
-          path: 'new-disabilities/unemployment-status',
+        newUnemployabilityFollowUp: {
+          title: 'Unemployability Choice',
```

Comment:
'Unemployment Status' instead of 'Unemployability Choice' to mirror the question and line them both up on the review page?